Unix/http/httpclient.c

```diff
@@ -813,6 +817,37 @@ static Http_CallbackResult _ReadData(
         _WriteTraceFile(ID_HTTPCLIENTRECVTRACEFILE, (char*)(handler->recvPage + 1), handler->receivedSize);
     }
 
+
+    if (!handler->ssl && handler->encrypting)
+    {
+
+#if ENCRYPT_DECRYPT
```

Comment:
The conditional is a protection requested by Paul to allow us to deactivate the encrypt code if needed. 
Don't know why it would be needed, but there it is.